assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -76,10 +81,11 @@ function combineElements(state, { firstId, firstElement, secondId }) {
 
   const {
     resource,
-    scale = 100,
-    focalX = 50,
-    focalY = 50,
-    isFill = false,
+    scale = DEFAULT_ATTRIBUTES_FOR_MEDIA.scale,
+    focalX = DEFAULT_ATTRIBUTES_FOR_MEDIA.focalX,
+    focalY = DEFAULT_ATTRIBUTES_FOR_MEDIA.focalY,
+    isFill = DEFAULT_ATTRIBUTES_FOR_MEDIA.isFill,
+    flip = {},
```

Comment:
What does empty `flip` mean?

Reply:
It's functionally the same as `flip: { vertical: false, horizontal: false }`, just simpler. Would you prefer it more explicit?

Reply:
Changed in dcc1ea85e (but essentially still the same).

Reply:
Ok. I guess it's ok. Just wanted to clarify. Didn't realize we allowed an empty object for that.